components/director/pkg/http/error_transport_test.go

```diff
@@ -7,10 +7,9 @@ import (
 	"net/http"
 	"testing"
 
-	"github.com/kyma-incubator/compass/components/system-broker/pkg/http/httpfakes"
```

Comment:
Why are a bunch of system-broker files moved to director component?

Reply:
They are used both in director and in system broker and director is the component to be used for shared packages.